tracegen/internal/tracegen/config.go

```diff
@@ -54,6 +75,9 @@ func (c *Config) Flags(fs *flag.FlagSet) {
 	fs.StringVar(&c.Endpoint, "otlp-endpoint", "localhost:4317", "Target to which the exporter is going to send spans or metrics. This MAY be configured to include a path (e.g. example.com/v1/traces)")
 	fs.BoolVar(&c.Insecure, "otlp-insecure", false, "Whether to enable client transport security for the exporter's grpc or http connection")
 	fs.BoolVar(&c.UseHTTP, "otlp-http", false, "Whether to use HTTP exporter rather than a gRPC one")
+
+	// custom headers
+	fs.Var(&c.Headers, "otlp-headers", `Custom Headers to be passed along with each otlp request. The value expected is a key(string), valued(string) pair json string (e.g '{"key1":"value1", "key2": "value2"}')`)
```

Comment:
```suggestion
	fs.Var(&c.Headers, "otlp-headers", `Custom headers to be passed along with each OTLP request. The value expected is a key(string), valued(string) pair json string (e.g '{"key1":"value1", "key2": "value2"}')`)
```

Reply:
Question: can we do it differently, like:


```
tracegen -otlp-header key1=value1 -otlp-header key2=value2
```

Reply:
yes! thats a great idea! How about `tracegen -otlp-header key1=value1,key2=value2` ? a comma seperated value so that flag doesn't have to be there over and over again ? Please let me know

Reply:
The repeated flag is a commonly used pattern in other tools, I think I'd prefer it that way.

Reply:
Actually @jpkrohling I was giving it a thought and since = is not a reserved character in either key or value of header, we may run into issues parsing for the right separator. Any ideas on what will be a right separator for key and value?

Reply:
Thanks @jpkrohling I have updated the PR with your suggestion. The test cases have been updated in the main description as well.  `tracegen -otlp-header key1=value1 -otlp-header key2=value2` is the new way. 

That said I have one question. Can you confirm if `=` cannot be used in header key strings? (reserved) `key=value` format works only if this premise is true. Please confirm (i have tried searching around but didn't get a definite answer). if it is not reserved can you please suggest me a good sperator

Reply:
Made the changes to use `-otlp-header key1=value1,key2=value2`